### PR TITLE
chore: refactor desktop account navigation

### PR DESCRIPTION
**Pull request by ivan-kalachikov**
### Jira-link:
https://virtocommerce.atlassian.net/browse/VCST-1874
### Artifact URL:
https://vc3prerelease.blob.core.windows.net/packages/vc-theme-b2b-vue-2.8.0-pr-1359-c70d-c70d68b6.zip

### CODE REVIEW

**Review by NaMax66 — APPROVED** [review]

File: client-app/shared/account/components/account-navigation-link-components/link-orders.vue [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<template>
+  <AccountNavigationLink :item="item">
+    <template v-if="isOrdersPage">
+      <div
+        v-for="facet in facets"
```
Comment (goldenmaya) [inline client-app/shared/account/components/account-navigation-link-components/link-orders.vue:5]: links within a link is weird, this needs to be changed

File: client-app/shared/account/components/account-navigation-link-components/link-lists.vue [hunk 2]
```diff
@@ -0,0 +1,45 @@
+<template>
+  <AccountNavigationLink :item="item">
+    <template v-if="isListDetails">
+      <div
+        v-for="list in lists"
```
Comment (goldenmaya) [inline client-app/shared/account/components/account-navigation-link-components/link-lists.vue:5]: links within a link is weird, this needs to be changed

Comment (ivan-kalachikov) [reply client-app/shared/account/components/account-navigation-link-components/link-lists.vue:5]: renamed to AccountNavigationItem for now

**Review by goldenmaya — APPROVED** [review]

File: client-app/core/composables/useNavigations.ts [hunk 3]
```diff
@@ -31,7 +31,13 @@ const desktopMainMenuItems = computed<ExtendedMenuLinkType[]>(() =>
 );
 
 const desktopAccountMenuItems = computed<ExtendedMenuLinkType | undefined>(() => {
-  return menuSchema.value ? getTranslatedMenuLink(menuSchema.value?.header?.desktop?.account) : undefined;
+  const schema = menuSchema.value?.header?.desktop?.account
+    ? _.clone(getTranslatedMenuLink(menuSchema.value.header.desktop.account))
```
Comment (Andrew-Orlov) [inline client-app/core/composables/useNavigations.ts:35]: Import only `clone` method from `lodash`.

**Review by Andrew-Orlov — APPROVED** [review]

**Pull request merged**